icu4j/main/classes/core/src/com/ibm/icu/number/FormattedNumber.java

```diff
@@ -28,6 +28,10 @@
     final DecimalQuantity fq;
     final MeasureUnit outputUnit;
 
+    // Grammatical gender of the formatted result. Not owned: must point at
+    // static or global strings.
```

Comment:
Remove, `Not owned: must point at    // static or global strings. `this is related to C++.

Reply:
done